mesonbuild/cmake/common.py

```diff
@@ -111,22 +119,22 @@ def log(self) -> None:
                 mlog.log(i)
 
 class CMakeTarget:
-    def __init__(self, data: dict):
-        self.artifacts = data.get('artifacts', [])
-        self.src_dir = data.get('sourceDirectory', '')
-        self.build_dir = data.get('buildDirectory', '')
-        self.name = data.get('name', '')
-        self.full_name = data.get('fullName', '')
-        self.install = data.get('hasInstallRule', False)
-        self.install_paths = list(set(data.get('installPaths', [])))
-        self.link_lang = data.get('linkerLanguage', '')
-        self.link_libraries = _flags_to_list(data.get('linkLibraries', ''))
-        self.link_flags = _flags_to_list(data.get('linkFlags', ''))
-        self.link_lang_flags = _flags_to_list(data.get('linkLanguageFlags', ''))
-        # self.link_path = data.get('linkPath', '')
-        self.type = data.get('type', 'EXECUTABLE')
-        # self.is_generator_provided = data.get('isGeneratorProvided', False)
-        self.files = []
+    def __init__(self, data: T.Dict[str, T.Any]) -> None:
+        self.artifacts               = data.get('artifacts', [])                            # type: T.List[str]
+        self.src_dir                 = data.get('sourceDirectory', '')                      # type: str
+        self.build_dir               = data.get('buildDirectory', '')                       # type: str
+        self.name                    = data.get('name', '')                                 # type: str
+        self.full_name               = data.get('fullName', '')                             # type: str
+        self.install                 = data.get('hasInstallRule', False)                    # type: bool
+        self.install_paths           = list(set(data.get('installPaths', [])))              # type: T.List[str]
+        self.link_lang               = data.get('linkerLanguage', '')                       # type: str
+        self.link_libraries          = _flags_to_list(data.get('linkLibraries', ''))        # type: T.List[str]
+        self.link_flags              = _flags_to_list(data.get('linkFlags', ''))            # type: T.List[str]
+        self.link_lang_flags         = _flags_to_list(data.get('linkLanguageFlags', ''))    # type: T.List[str]
+        # self.link_path             = data.get('linkPath', '')                             # type: str
+        self.type                    = data.get('type', 'EXECUTABLE')                       # type: str
+        # self.is_generator_provided = data.get('isGeneratorProvided', False)               # type: bool
+        self.files                   = []                                                   # type: T.List[CMakeFileGroup]
```

Comment:
I'm not crazy about lining up `=` signs like this, I don't think it improves readability (as opposed to say lining up data in lists of tuples or dicts), but... I don't care too much so if you like it leave it.

Reply:
It really improved the readability for me though. Not aligning is not big deal for one or two assignments, but helps a lot if there is a huge block.

I guess this is just a difference in taste :)